network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/AbstractBranchImpl.java

```diff
@@ -110,14 +110,21 @@ public void setCurrentLimits(Branch.Side side, CurrentLimitsAttributes currentLi
         if (side == Branch.Side.ONE) {
             CurrentLimitsAttributes oldCurrentLimits = resource.getAttributes().getCurrentLimits1();
             resource.getAttributes().setCurrentLimits1(currentLimits);
+            updateResource();
             index.notifyUpdate(this, "currentLimits1", oldCurrentLimits, currentLimits);
         } else if (side == Branch.Side.TWO) {
             CurrentLimitsAttributes oldCurrentLimits = resource.getAttributes().getCurrentLimits2();
             resource.getAttributes().setCurrentLimits2(currentLimits);
+            updateResource();
             index.notifyUpdate(this, "currentLimits2", oldCurrentLimits, currentLimits);
         }
     }
 
+    @Override
+    public AbstractIdentifiableImpl getObject() {
```

Comment:
getIdentifiable ?
getObjectIdentifiable ?
Is it just for the `leg` which is not a identifiable

Reply:
it's for currentlimitsholder

Reply:
Renamed to getIdentifiable